### PR TITLE
updated seller info

### PR DESCRIPTION
**Pull request by Arya23sharma**
new code,excluded pakagelock.json

### CODE REVIEW

File: shopkeeper/src/components/seller-info.component.js [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+      pin: '',
+      address1 : '',
+      address2:'',
+      city: '', 
+      state: '',
+      fields: '',
+      errors: '',
+      
+    }
+  //   this.handleSubmit = this.handleSubmit.bind(this);
```
Comment (Adarshg315) [inline shopkeeper/src/components/seller-info.component.js:26]: remove the unwanted comment always

File: shopkeeper/src/components/seller-info.component.js [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+    this.address1 = this.address1.bind(this);
+    this.address2 = this.address2.bind(this);
+    this.city = this.city.bind(this);
+    this.check = this.check.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    }
+     handleSubmit = (event) => {
+         event.preventDefault()
+         console.log(event.target[0].value)
+      //   console.log(event.target.elements.firstname.value)
```
Comment (Adarshg315) [inline shopkeeper/src/components/seller-info.component.js:81]: remove this too

**Review by Adarshg315 — COMMENTED** [review]
can you send me the working demo screen recorded vidio

File: shopkeeper/src/components/seller-info.component.js (hunk 2 above)
Comment (Arya23sharma) [reply shopkeeper/src/components/seller-info.component.js:81]: ohk

**Pull request closed without merging**